libdnf/base/base.cpp

```diff
@@ -97,8 +97,11 @@ void Base::add_plugin(plugin::IPlugin & iplugin_instance) {
 }
 
 void Base::load_plugins() {
-    if (const char * plugin_dir = std::getenv("LIBDNF_PLUGIN_DIR")) {
-        plugins.load_plugins(plugin_dir);
+    if (const char * plugins_config_dir = std::getenv("LIBDNF_PLUGINS_CONFIG_DIR")) {
+        plugins.load_plugins(plugins_config_dir);
```

Comment:
We've already discussed this, but I think we both forgot a bit. It would be better to load the plugins by listing the plugin directory itself. Just to prevent confusion in case of inconsistencies, like a user putting an .so of the plugin into the plugin directory and have it silently ignored.

Then there are a couple of ways to handle the config file. I personally wouldn't be opposed to not have a mandatory config file. In case it would be missing, it would mean the plugin is enabled by default. The advantage is simpler design for simple plugins, disadvantage is the user must create the actual file themselves if it's missing and figure out the contents to e.g. set `enabled = false`.

Or we could have a mandatory config with a mandatory `enabled = ...` option in it. I think you prefer this way, I'll leave that to you.

The PR looks good otherwise.

Reply:
> Or we could have a mandatory config with a mandatory enabled = ... option in it. I think you prefer this way, I'll leave that to you.

I prefer to have mandatory configuration file. A plugin without configuration file will not be loaded.